Create yasmine.json

Yasmine091 opened the pull request:
```
// ====================== [BRANCH MERGED] ====================== \\
//                       THANK YOU SO MUCH!                      \\
// ====================== [BRANCH MERGED] ====================== \\
```

Yasmine091 committed:
Create yasmine.json

Yasmine091 committed:
Update yasmine.json

anime-oni commented:
Hey @Yasmine091,

https://docs.is-a.dev/faq/#who-can-use-ns-records

Yasmine091 committed:
Update yasmine.json

anime-oni commented:
Also btw checkout https://docs.is-a.dev/guides/netlify/

Yasmine091 committed:
Update yasmine.json

Yasmine091 commented:
> Also btw checkout https://docs.is-a.dev/guides/netlify/

Thanks its done, i used the A record as the guide stated

anime-oni commented on yasmine.json:
```diff
@@ -0,0 +1,9 @@
+        "email": "y.contact@malaxit.eu"
+    },
+    "records": {
+        "A" : "75.2.60.5"
```
Add `[]`s ie:

`"A" : "75.2.60.5"` —> `"A" : ["75.2.60.5"]`

Simple errors dw!

Yasmine091 replied:
Thanks!!!

Yasmine091 committed:
Update yasmine.json

Yasmine091 committed:
Merge branch 'main' into patch-1

Yasmine091 committed:
Merge branch 'main' into patch-1

Yasmine091 committed:
Update yasmine.json

Netlify is very limited and my site went down, so i had to migrate back to my hosting provider

Yasmine091 commented:
I ended up moving from netlify back to my old hosting provided as it's very limited (the website already went down just because of some theme updates lol) so the URL to visit and the A record have been changed to the hosting provider

STICKnoLOGIC reviewed (dismissed):
lgtm

Yasmine091 committed:
Merge branch 'main' into patch-1

Yasmine091 committed:
wrong folder

Yasmine091 committed:
wrong folder

Yasmine091 committed:
Merge branch 'main' into patch-1

Yasmine091 committed:
Merge branch 'main' into patch-1

Yasmine091 committed:
Merge branch 'main' into patch-1

Yasmine091 committed:
Merge branch 'main' into patch-1

Yasmine091 committed:
Merge branch 'main' into patch-1

Yasmine091 committed:
Merge branch 'main' into patch-1

Yasmine091 committed:
Merge branch 'main' into patch-1

Yasmine091 committed:
Merge branch 'main' into patch-1

Yasmine091 commented:
I moved it to /domains
Needs review @STICKnoLOGIC

Yasmine091 reviewed (commented):
The file has been moved to domains/

Yasmine091 committed:
Merge branch 'main' into patch-1

Yasmine091 committed:
Merge branch 'main' into patch-1

Yasmine091 committed:
Merge branch 'main' into patch-1

Yasmine091 committed:
Merge branch 'main' into patch-1

Yasmine091 committed:
Merge branch 'main' into patch-1

Yasmine091 committed:
Merge branch 'main' into patch-1

Yasmine091 committed:
Merge branch 'main' into patch-1

Yasmine091 committed:
Merge branch 'main' into patch-1

Yasmine091 committed:
Merge branch 'main' into patch-1

Yasmine091 committed:
Merge branch 'main' into patch-1

Yasmine091 committed:
Merge branch 'main' into patch-1

STICKnoLOGIC reviewed (approved):
lgtm
